plugins/c8y_configuration_plugin/src/main.rs

```diff
@@ -78,10 +71,10 @@ async fn create_mqtt_client(mqtt_port: u16) -> Result<mqtt_channel::Connection,
         mqtt_channel::TopicFilter::new_unchecked(C8yTopic::SmartRestRequest.as_str());
     topic_filter.add_all(health_check_topics("c8y-configuration-plugin"));
 
-    topic_filter.add_all(ResponseTopic::ConfigSnapshotRequest.into());
-    topic_filter.add_all(ResponseTopic::ConfigSnapshotResponse.into());
-    topic_filter.add_all(ResponseTopic::ConfigUpdateRequest.into());
-    topic_filter.add_all(ResponseTopic::ConfigUpdateResponse.into());
+    topic_filter.add_all(ConfigResponseTopic::SnapshotRequest.into());
```

Comment:
This plugin doesn't need to **subscribe** to the `SnapshotRequest` and `UpdateRequest` topics as this plugin is the one **publishing** these **requests** to those topics. So, it just needs to subscribe to the **responses** from the child devices. It's the child devices that will have to subscribe to those **request** topics.